ripple/src/main/webapp/businessdetails.js

```diff
@@ -84,12 +89,61 @@ function newReview() {
     localStorage.setItem("reviewText", reviewText);
     localStorage.setItem("redirectAfterSignIn", "businessdetails.html"); 
     window.location = "login.html";
-  } else {
-    location.reload();
+  } else { // Community member
+    // 1. Add uid into 'businessDetails' colection 'usersReviewed'
+    // 2. Initialize 'reviews' doc with the 'uid' as doc id. 
+    // Update star rating, price rating, timestamp, reviewText
+    // Set everything with likes / dislikes to 0
+    updateUsersReviewedList(uid);
+    createReviewDocument(uid, reviewText);
+    // location.reload();
```

Comment:
nit:remove?

Reply:
Will be keeping it for now because it may be needed when we implement deleting comments